src/activities/system.clj

```diff
@@ -14,6 +14,8 @@
             [buddy.auth.middleware :refer [wrap-authentication]]
             [buddy.auth.backends :as backends]))
 
+(require 'activities.time)
+
```

Comment:
Is there a reason for requiring it outside the `ns` declaration?

Reply:
yes. it's being required for side-effects, so it's not actually used in this namespace. This means that some tools like `clj-refactor` might think it's not used and thus remove it from the `ns` form. doing it this way makes it clear that it's loaded for side effects and prevents that problem.